server/src/servershot.js

```diff
@@ -387,45 +389,30 @@ class ServerClip extends AbstractShot.prototype.Clip {
 
 Shot.prototype.Clip = ServerClip;
 
-Shot.get = function(backend, id, deviceId, accountId) {
-  return Shot.getRawValue(id, deviceId, accountId).then((rawValue) => {
-    if (!rawValue) {
-      return null;
-    }
-    const json = JSON.parse(rawValue.value);
-    const jsonTitle = json.userTitle || (json.openGraph && json.openGraph.title) || json.docTitle;
-    json.docTitle = jsonTitle || rawValue.title;
-    if (!json.url && rawValue.url) {
-      json.url = rawValue.url;
-    }
-    const shot = new Shot(rawValue.userid, backend, id, json);
-    shot.urlIfDeleted = rawValue.url;
-    shot.accountId = rawValue.accountId;
-    shot.expireTime = rawValue.expireTime;
-    shot.deleted = rawValue.deleted;
-    shot.blockType = rawValue.blockType;
-    return shot;
-  });
-};
-
-// FIXME: What is the difference between Shot.getFullShot and Shot.get?
-Shot.getFullShot = function(backend, id) {
+Shot.get = async function(backend, id, deviceId, accountId) {
   if (!id) {
-    throw new Error("Empty id: " + id);
+    return null;
```

Comment:
Why this change?

Reply:
we are handling null return in calling functions, reverted back as we can keep throw error here as it does no harm.